spec/bookmarks-view-spec.coffee

```diff
@@ -114,43 +115,48 @@ describe "Bookmarks package", ->
         atom.commands.dispatch editorElement, 'bookmarks:toggle-bookmark'
         expect(bookmarkedRangesForEditor(editor).length).toBe 0
 
-    it "toggles proper classes on proper gutter row", ->
+    it "toggles proper classes on proper gutter and line row", ->
       editor.setCursorBufferPosition([3, 10])
       expect(editorElement.shadowRoot.querySelectorAll('.bookmarked').length).toBe 0
 
       atom.commands.dispatch editorElement, 'bookmarks:toggle-bookmark'
       lines = []
 
       waitsFor ->
-        lines = editorElement.shadowRoot.querySelectorAll('.bookmarked')
+        editorElement.shadowRoot.querySelectorAll('.line.bookmarked').length is 1
```

Comment:
This line doesn't seems to do anything. Same goes for a few more below. 
To [my understanding](http://js2.coffee/#coffee/try:waitsFor%20-%3E%0A%20%20editorElement.shadowRoot.querySelectorAll(%27.line.bookmarked%27).length%20is%201%0A%20%20lines%20%3D%20editorElement.shadowRoot.querySelectorAll(%27.line-number.bookmarked%27)%0A%20%20lines.length%20is%201) CoffeeScript returns the value from the last line within a function, defining two (or more) lines will still only return the last line.